src/frontend/src/pages/ProjectDetailPage/ProjectEdit/ProjectEditContainer.tsx

```diff
@@ -202,10 +203,25 @@ const ProjectEditContainer: React.FC<ProjectEditContainerProps> = ({ project, ex
           setProjectLead={setProjectLeadId}
           setProjectManager={setprojectManagerId}
         />
-        <PageBlock title="Links">
-          <LinksEditView watch={watch} ls={links} register={register} append={appendLink} remove={removeLink} />
+        <PageBlock title="Project Summary">
```

Comment:
remove this summary and the pageblock around it, looks like this rn: 
<img width="905" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/9bbcb6ff-fa36-4254-b466-85dffd719634">